src/lib/operations.ts

```diff
@@ -190,6 +190,114 @@ const prepareToEditOffsetPlane: PrepareToEditCallback = async ({
   }
 }
 
+const prepareToEditHelix: PrepareToEditCallback = async ({ operation }) => {
+  const baseCommand = {
+    name: 'Helix',
+    groupId: 'modeling',
+  }
+  if (operation.type !== 'StdLibCall' || !operation.labeledArgs) {
+    return baseCommand
+  }
+
+  // TODO: find a way to loop over the arguments while keeping it safe
```

Comment:
Input welcome here. Lots of arguments here. 4 kcl, 2 options

Reply:
Interesting! Great test case for making this code more concise with helper functions and reorganization, thanks for doing this. I'm going to see if we can make TS happy with an `Array.every` over all the keys we need to be present in `operation.labeledArgs`, then try to loop over them in one go.

Reply:
Oh yeah bet, I half-ass tried last week and gave up, we'll improve it elsewhere